addon/components/document-view.js

```diff
@@ -41,7 +41,7 @@ export default class DocumentViewComponent extends Component {
       this.sort = sortAttribute;
       this.sortDirection = "";
     }
-    this.router.transitionTo({
+    this.router.transitionTo(this.router.currentRouteName, {
```

Comment:
Is this an engine specific fix or are `queryParam` only transitions deprecated?

Reply:
Engine router specific: https://github.com/villander/ember-engines-router-service/issues/74